src/patternfly/components/Popover/examples/Popover.md

```diff
@@ -132,34 +140,126 @@ import './Popover.css'
 {{/popover}}
 ```
 
+### Variations
```

Comment:
Can you create unique example names for these variations? A default "Icon" example with "Success alert", "Warning alert", etc.

Reply:
This is fixed now. Please advise if the names are fine. I can still change them.